### ISSUE TITLE
Reproducing Polynomial Coefficients

### ISSUE THREAD

**Issue by oloeopia** [shibatch/sleef#258]
Would it be possible to get the settings used for each of the gencoef invocations in order to reproduce each of the polynomials?

**Comment by shibatch** [comment]
For most of them, it is basically possible.
Because it involves randomized optimization, the generated coefficients are not bitwise identical.

**Comment by shibatch** [comment]
Please tell me which functions you want to regenerate coefficients for, if you have difficulty in doing so.

**Comment by shibatch** [comment]
The settings are included in dp.h and sp.h.

**Comment by oloeopia** [comment]
Okay, now I see, I was misinterpreting how to use gencoef. I think I can make it work, but if I find one that I can't reproduce an equivalent polynomial with the same error bounds, I will let you know.

**Comment by shibatch** [comment]
For some functions, you need multiple trials.

**Comment by oloeopia** [comment]
If you used a deterministic random number generator and saved the seeds that worked, wouldn't you be able to ensure that you get bitwise identical results?

**Comment by shibatch** [comment]
Yes, it’s possible. I didn’t do that because I didn’t see a demand by users for generating bitwise identical results.
